### ISSUE TITLE
Translate(/search) recently added [$200]

### ISSUE THREAD

**Issue by Kingdutch** [WeblateOrg/weblate#2455]
We are just starting with translations for some of our projects which causes us to have quite a backlog of untranslated strings. One of the things we'd like to do is translate the most recently added strings (because they'll be most visible).

In search we can specify "Changed since" but this seems to work mostly on the translations.

Is there a way to order missing translations by the date on which their source string was added? Or alternatively to show only missing translations for source strings that were added after a certain date?

**Comment by nijel** [comment]
Currently it's not possible, but it might be useful.

**Comment by nijel** [comment]
You can use [Bountysource](https://www.bountysource.com/issues/67291562-question-translate-search-recently-added) for that :-). Other options for getting paid support are listed on https://weblate.org/support/

**Comment by nijel** [comment]
In the end, this might be blocked by #2831, so the question is whether to move all this to 3.10 or prioritize blocking issues for 3.9.

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see a problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
